.travis.yml

```diff
@@ -26,6 +26,9 @@ env:
   - BUILD=site
   - BUILD=sonar
 
+matrix:
+  fast_finish: true
```

Comment:
You are right... it was promised for 2017Q1, but seems never delivered https://github.com/travis-ci/travis-ci/issues/1066#issuecomment-257441420